.dockerignore

```
@@ -0,0 +1,3 @@
+node_modules
+dist
+.cache-loader
```

```suggestion
.cache-loader
.husky/_/
```
We can safely remove `.husky/_/` as well.
```
❯ g clean -fxd                                                                                                                                                              odf-console/git/master 
Removing .cache-loader/
Removing .husky/_/
Removing dist/
Removing node_modules/
```

---

ack